src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String SPLITTER = ",";
+    private static final String SUPPLY = "supply";
+    private static final int STEP_COUNT = 2;
 
+    public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
This method has too much responsibilities, I suggest to create 3 methods: readFromFile, wirteToFile, collectStatistics and then in getStatistic method call these 3

Reply:
agree, created